### DISCUSSION TITLE
Need a Way to Dismiss the Pre-KI Chat UI

### DISCUSSION THREAD

**Discussion by patmauro** [H-uru/Plasma#1519] [Ideas & Proposals]
Imagine you're a new player checking out URU for the first time. You select the "New Player" option, watch the intro, and walk through the desert towards Zandi. As a side-effect of the new subtitles feature, by default, Zandi's dialogue now appears in a chat window that has just appeared on the screen. Great - very useful! Except... you slowly realize this window does not fade, and there's no way to dismiss it - so now, as far as you can tell, this text is now stuck on your screen permanently, with no way to get rid of it.

Experienced players will know they can type `/clearchat` to make this go away, but a _new_ player, who is just getting used to the game, will not know to to this. So - how can we help our new players out and make things a little more intuitive?

The main issue here is that the standard KI UI can be opened & dismissed at will - Opening & Closing the KI will dismiss the UI, plus there is a physical button on the UI that can be used to immediately close the UI - but **no such feature exists** in the Pre-KI Chat UI. 

Assuming there's there's _hypothetically_ a way this can actually be enabled in the pre-KI UI (without clearing chat), I see a couple possible ways to integrate this into the UX, none of which are mutually exclusive:

1. Add a "Close UI" UI element similar to the KI interface. This can be a simple "X" and can even be in the same approximate location as the normal KI's "Close UI" button.
![image](https://github.com/H-uru/Plasma/assets/6642931/a0688153-cff6-4fb9-9df5-2c18e3e97c69)
2. Pressing the pre-KI ">CHAT" menu button engages "start chat," but there is actually no such option in _normal_ gameplay, so this sort of feels like an unnecessary feature. Instead, I would recommend this button be repurposed to toggle SHOW/HIDE of the chat UI. 
![image](https://github.com/H-uru/Plasma/assets/6642931/593a6a67-0974-42f7-a3b0-b4bfe2fae8cd)
3. F2 (or more specifically, the "Open KI" command that is mapped to this button by default) could also Show/hide the chat UI in pre-KI contexts. However, I assume this would probably be a lot more complicated, as we're talking about re-routing a normal KI command to behave differently in this a specific context. If the other two options were in place, this option would probably be not needed - but this might be a helpful adaptation either way.

Thoughts?

**Comment by Deledrius** [comment]
I definitely think options 2 & 3 sound like good enhancements to fix this UX issue.  They feel like things that should work that way.
